group/src/cofactor.rs

```diff
@@ -17,10 +17,16 @@ pub trait CofactorGroup:
     /// If `Self` implements `PrimeGroup`, then `Self::Subgroup` may be `Self`.
     type Subgroup: PrimeGroup<Scalar = Self::Scalar> + Into<Self>;
 
-    /// Returns `[h] self`, where `h` is the cofactor of the group.
+    /// Maps `self` to the prime-order subgroup by clearing any torsion component.
```

Comment:
"any torsion component" is incorrect technically, you could say h-torsion or just merge this sentence with the one afterward by saying

```
/// Maps `self` to the prime-order subgroup by multiplying this element by some `k`-multiple of the cofactor.
///
/// The value `k` does not vary between inputs for a given implementation, but may vary between different implementations of `CofactorGroup` because some groups have more efficient methods of clearing the cofactor when `k` is allowed to be different than `1`.
```

modulo word wrap